backend/src/database/users.go

```diff
@@ -29,24 +29,29 @@ func (db *DB) GetCurrentUsers(page, itemsPerPage int, facilityId uint, order str
 	offset := (page - 1) * itemsPerPage
 	var count int64
 	var users []models.User
-	if err := db.Model(&models.User{}).Where("facility_id = ?", facilityId).Count(&count).Error; err != nil {
-		return 0, nil, newGetRecordsDBError(err, "users")
-	}
-	tx := db.Model(&models.User{})
+
+	tx := db.Model(&models.User{}).Where("facility_id = ?", facilityId)
 	switch role {
 	case "admin":
-		tx = tx.Where("role IN ('admin', 'system_admin')")
```

Comment:
this needs to stay the way it was, otherwise it will not work for the system_admin user